### ISSUE TITLE
KeePassXC-Browser with 2.5.4 - Does not match domain

### ISSUE THREAD

**Issue by andynz22** [keepassxreboot/keepassxc#4749]
I found that using  KeePassXC-Browser Version: 1.6.3 with KeePassXC Version: 2.5.4 did not show stored login prompts for stored domains.

Example stored credentials for "https://www.aliexpress.com" shows in KeePassXC Version: 2.5.4 but did not show in KeePassXC-Browser Version: 1.6.3.

I downgraded to KeePassXC Version: 2.5.1 and when logging in at "https://www.aliexpress.com" pop up login I was able to use the stored credentials.

**Comment by droidmonkey** [comment]
There were several changes between 2.5.0 and 2.5.3 to fix browser domain matching. The current state works, you have to make sure if you are connecting to an http:// site that you put that in.

**Comment by andynz22** [comment]
Hmm.  I just went through all the latest versions and found the following with 1.6.3 installed on W10 Firefox 76.0.1 (64-bit) and aliexpress.com pop up login window as the example:
KeePassXC-2.5.1-Win64.msi - shows 1 entry for login credentials and populates.
KeePassXC-2.5.2-Win64.msi - no entry shown
KeePassXC-2.5.3-Win64.msi - no entry shown
KeePassXC-2.5.4-Win64.msi - no entry shown

**Comment by droidmonkey** [comment]
Is the website address in the browser bar exactly https://www.aliexpress.com? Is the login prompt being displayed in an iframe?

**Comment by andynz22** [comment]
Yes exactly https://www.aliexpress.com/ which matches the database entry.
Login is iframe pop up yes.

**Comment by droidmonkey** [comment]
Then you need to match the domain of the iframe

**Comment by andynz22** [comment]
I think the iframe was a red hearing. As I say, vs 2.5.1 works ok but not later versions.

I did some further testing as, whilst I was seeing the issue, obviously others are not. As always there is more information on the issue in the background.  My password database had been on a bit of a tiki tour over the last 12 months. Imported to Bitwarden from lastpass, I then imported the Bitwarden into Keepass 2 using the included bitwarden plugin.

The problem I was seeing related to a number of domains I had multiple credentials for. Showing one option in the browser when multiple existed in the Keepassxc database.
I tried renterring those credentials manually into Keepassxc as new entries.  Having done that they all showed in the browser.

Sl the fault seems to be in the import process. Credentials display as ok and when you search in Keepassxc they all show but not in the browser addon.

I will try a direct import from bitwarden into keepaasxc and see if that works.
